inbm/Changelog.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
  - RTC 536078 - Added package list option to inbc, cloud, and internal manifest. This allows SOTA to run an install/upgrade command on a set of individual packages rather than all installed packages.
+ - RTC  536910 - [source] Remove ubuntuAptSource configuration tag and underlying code
```

Comment:
highlight this is a potentially breaking change and anyone relying on ubuntuAptSource needs to use the new 'source' mechanism

Reply:
@tsirlapu i see you marked the conversation as resolved, but I don't see that you're highlighting this as a breaking change

Reply:
@gblewis1 how does this look

Reply:
@tsirlapu maybe something like `- (BREAKING CHANGE) RTC  536910 - [source] Remove ubuntuAptSource INBM configuration tag and underlying code; replaced with source command`